### PR TITLE
Pass urlencoded query parameters to download button call

### PR DESCRIPTION
**Pull request by Donnype**
### Changes
Use the same filters (severity) for the pdf report on the findings list page.

### Issue link
Closes https://github.com/minvws/nl-kat-coordination/issues/1631

### Proof
Filter on medium:
image

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/rocky/templates/partials/findings_list_toolbar.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <a href="{% url 'finding_add' organization.code %}" class="button ghost">{% translate "Add finding" %}</a>
     <a href="{% url "finding_type_add" organization.code %}"
        class="button ghost">{% translate "Add finding type" %}</a>
-    <a href="{% url "findings_pdf_report" organization.code %}"
+    <!-- Note that we cannot pass e.g. severity=request.GET.severity explicitly since this is a list parameter -->
+    <a href="{% url "findings_pdf_report" organization.code %}?{{ request.GET.urlencode }}"
+       sdf
```
Comment (ammar92) [inline rocky/rocky/templates/partials/findings_list_toolbar.html:9]: Typo? :)
```suggestion
```

Comment (Donnype) [reply rocky/rocky/templates/partials/findings_list_toolbar.html:9]: A random keystroke appeared

**Review by ammar92 — APPROVED** [review]

**Comment by underdarknl** [comment]
We do need to show In the report which flags where enabled when creating this query. The query could be tuned to exclude relevant data, and if the report does not explain this to the reader the reader might take the partial report to be a complete report by mistake.
Ideally, We should include all relevant flags (language, timetamp of creation, timestamp of requested data, applied filters and organization) in the footnote of the report, so that someone has enough data to recreate the report and validate the results.

**Comment by Donnype** [comment]
@underdarknl Do we really want to use the footnote or can I add it to the summary?

Schermafbeelding 2023-08-31 om 14 21 23

**Review by underdarknl — APPROVED** [review]

**Review by Rieven — APPROVED** [review]

**Pull request merged**